RecoParticleFlow/PFProducer/python/pfBasedElectronPhotonIso_cff.py

```diff
@@ -1,8 +1,6 @@
 import FWCore.ParameterSet.Config as cms
 
 from CommonTools.ParticleFlow.pfParticleSelection_cff import *
-#from RecoParticleFlow.PFProducer.electronPFIsolationDeposits_cff import *
```

Comment:
I don't know the original reason why these electron isolation deposits were not considered.
On the other hand, the config is called ```pfBasedElectronPhotonIso_cff.py``` and probably letting them commented out but still in the config will help developers remind that only photon isolation deposits are considered by now.